src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 0.3s, transform 0.3s ease;
```

Comment:
Requirements:
use SCSS variables for values, used in several places throughout the code